src/app.js

```diff
@@ -1,40 +1,36 @@
 import express from 'express';
-import path from 'path';
-import logger from 'morgan';
+import morgan from 'morgan';
 import bodyParser from 'body-parser';
+import cookieParser from 'cookie-parser';
+import expressSession from 'express-session';
+import helmet from 'helmet';
 import routes from './routes';
+import { auth } from './auth';
 
 const app = express();
-app.disable('x-powered-by');
 
-// View engine setup
-app.set('views', path.join(__dirname, '../views'));
-app.set('view engine', 'pug');
+app.use(helmet());
+app.use(morgan('combined'));
+app.use(cookieParser());
```

Comment:
The `cookieParser` and the `expressSession` are required for the twitter oauth?

Reply:
yes, express session is used to create a session cookie and cookie parser to parse that cookie.